src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -470,6 +470,27 @@ private void OnProcessExit(object sender, EventArgs e)
 
         // Synchronization primitive for controlling initialization of Default load context
         private static readonly object s_initLock = new Object();
+
+        // Occurs when an Assembly is loaded
+        public static event AssemblyLoadEventHandler AssemblyLoad 
+        { 
```

Comment:
AssemblyLoad should be static because of it is counterpart GetLoadedAssemblies that is also static. So static is the right choice for that one.

TypeLoaded is WinRT-specific. I think it is fine for WinRT types to be loaded globally - it is what we have in desktop; and making it to not to be the case makes it super hard to reason about interop. So static seems like the good choice for this one as well.

The only one that may be a bit questionable is the ResourceLoaded. I believe it is super rarely used, and it may be reasonable to have it global as well.